catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/CategoryListFragment.java

```diff
@@ -150,6 +150,10 @@
 			asList(R.string.formula_editor_function_collides_with_color, R.string.formula_editor_function_color_touches_color);
 	private static final List<Integer> OBJECT_COLOR_PARAMS =
 			asList(R.string.formula_editor_function_collides_with_color_parameter, R.string.formula_editor_function_color_touches_color_parameter);
+	private static final List<Integer> OBJECT_COLOR_TOUCHES_COLOR =
```

Comment:
There is no need to make new variables here, as they are already contained in `OBJECT_COLOR_COLLISION` and `OBJECT_COLOR_PARAMS`.